os-server/src/main/java/app/openschool/course/discussion/peers/question/PeersQuestionServiceImpl.java

```diff
@@ -68,8 +75,15 @@ public void delete(Long questionId, Long enrolledCourseId, String currentUserEma
   }
 
   @Override
-  public Page<PeersQuestion> findQuestionByCourseId(Long enrolledCourseId, Pageable pageable) {
-    return peersQuestionRepository.findQuestionByEnrolledCourseId(enrolledCourseId, pageable);
+  public Page<PeersQuestion> findQuestionByCourseId(
+      Long enrolledCourseId, Pageable pageable, String searchQuery) {
+    if (searchQuery != null && searchQuery.length() < 3) {
```

Comment:
please use Objects.nonNull method, and move the validation logic into a separate private method

Reply:
make the number a constant